packages/zowe-explorer/src/job/actions.ts

```diff
@@ -541,3 +542,74 @@ export async function sortJobsBy(jobs: IZoweJobTreeNode, jobsProvider: IZoweTree
     });
     jobsProvider.refresh();
 }
+
+export async function filterJobs(jobsProvider: IZoweTree<IZoweJobTreeNode>): Promise<vscode.InputBox> {
+    let acutal_jobs;
+    let flag = false;
+    for (const level of jobsProvider.mSessionNodes) {
+        if (level.label === "zosmf") {
+            acutal_jobs = level.children;
+            if (level.collapsibleState === 1) {
+                vscode.window.showInformationMessage("Inorder to filter jobs,first populate them using search icon");
```

Comment:
- `Gui.infoMessage` can be used here
- Consider using localization for this message so it can be translated 🙂